_books/ion-1-1/src/modules/directives.md

```diff
@@ -163,3 +163,9 @@ When the version is not provided, a default value of 1 is used.
 // Defaults to version 1
 (:$ion use "com.example.geometry")
 ```
+
+## Ion 1.0 symbol table syntax not supported
+
+In Ion 1.1 streams, Ion 1.0 symbol table syntax (`$ion_symbol_table::{...}`) is not supported.
```

Comment:
And `$ion_symbol_table::null.struct`?

> If the struct is null (null.struct) then it is treated as if it were an empty struct.

I think that means it's treated as an empty symbol table struct, rather than an empty user value struct.